### ISSUE TITLE
Idea: Make Graveyard a package on Nimble.

### ISSUE THREAD

**Issue by juancarlospaco** [nim-lang/graveyard#2]
Make Graveyard a package on Nimble. `nimble install graveyard`
:smiley:

**Comment by Araq** [comment]
PRs are welcome.

**Comment by juancarlospaco** [comment]
Awesome :grey_exclamation: Graveyard is life :joy_cat:

**Comment by kaushalmodi** [comment]
`nimble install xmldom` and `nimble install xmldomparser` will already start working once https://github.com/nim-lang/packages/pull/885 is merged.

**Comment by juancarlospaco** [comment]
:open_mouth: :+1:

**Comment by juancarlospaco** [comment]
So it seems the work has been already done somewhere else, we can close this,
thanks for the participation everyone. :smiley_cat:

**Comment by kaushalmodi** [comment]
@juancarlospaco FYI, nimble installs of those 2 packages will work now.
